pkg/activator/handler/handler.go

```diff
@@ -189,6 +194,12 @@ func (a *activationHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		a.reporter.ReportResponseTime(namespace, serviceName, configurationName, name, httpStatus, duration)
 	})
 	if err != nil {
+		// Set error on our capacity waiting span and end it
+		ccWaitSpan.Annotate([]trace.Attribute{
+			trace.StringAttribute("activator.throttler.error", err.Error()),
+		}, "CapacityWait")
```

Comment:
It is not necessarily "Capacity Wait" it can be overload, or it can be breaker capacity overflow. So probably Just `Activation`?
But, I concur, that probably it would be better if we had two spans, one internal in `Try` to wait for the endpoint. And one external for the time to proxy.

Reply:
to me activation is waiting on specifically a pod to start up - I think both of the situations you describe are capacity related issues. I dunno, naming is a hard problem :/

Reply:
Yeah, that's why I say "TryRequest" or "Proxy" more generic and would fit :) But I guess it's a minor issue anyway.

Reply:
Or maybe we can call this ThrottlerWait rather than try and describe implementation?

Reply:
it's more than wait :) it's also `do`. So `Try` if what fails with error, so  `(Throttler)?Try` -- feels the closest to reality. 

👍 for bike-shedding!